platforms/documentation/docs/src/docs/userguide/authoring-builds/structuring/multi_project_configuration_and_execution.adoc

```diff
@@ -13,6 +13,30 @@
 // limitations under the License.
 
 [[configuration_and_execution]]
-= Configuration and Execution time
+= Configuration On Demand
 
-This page has been removed.
+Configuration-on-demand attempts to configure only relevant projects for requested tasks, i.e., it only executes the build script file of projects participating in the build.
+This way, the configuration time of a large multi-project build can be reduced.
+
+The configuration-on-demand feature is _incubating_, so not every build is guaranteed to work correctly.
+The feature should work well for multi-project builds that have decoupled projects.
```

Comment:
Is there a header link for decoupled projects? If so we can link to it

Reply:
Added.